src/analysis/rust_type.rs

```diff
@@ -241,11 +241,13 @@ pub fn parameter_rust_type(
     direction: library::ParameterDirection,
     nullable: Nullable,
     ref_mode: RefMode,
+    async: bool,
 ) -> Result {
     use library::Type::*;
     let type_ = env.library.type_(type_id);
     let rust_type = rust_type_full(env, type_id, nullable, ref_mode);
     match *type_ {
+        Fundamental(super::library::Fundamental::Pointer) if async => Ok("TODO4".to_string()), // FIXME: remove
```

Comment:
Wut?

Reply:
Yeah, some TODO that are not generated.
I'll have to figure out why.